docs/components/radio.md

```diff
@@ -1,161 +1,163 @@
+
+<div>
+  <div class="cards-container">
+    <KRadio
+      v-model="cardRadio"
+      description="Choose this option if you want your APIs to be publicly accessible by anyone on the internet."
+      is-card
+      label="Public"
+      selected-value="public"
+    >
+      <WorldIcon />
+    </KRadio>
+    <KRadio
+      v-model="cardRadio"
+      description="Choose this option if you want your APIs to only be accessible from within your private network."
+      is-card
+      label="Private"
+      selected-value="private"
+    >
+      <WorldPrivateIcon />
+    </KRadio>
+  </div>
+  <div>Selected: {{ cardRadio }}</div>
+</div>
 
-```html
+```vue
 <template>
-  <KRadio type="card" selected-value="foo" v-model="cardRadio">
-    <img src="/img/kong-logo.png" alt="Kong logo" />
-    <div class="k-radio-label">Foo</div>
-    <div class="k-radio-description">This subheader</div>
+  <KRadio
+    v-model="cardRadio"
+    description="Choose this option if you want your APIs to be publicly accessible by anyone on the internet."
+    is-card
+    label="Public"
+    selected-value="public"
+  >
+    <WorldIcon />
   </KRadio>
-  <KRadio type="card" selected-value="bar" v-model="cardRadio">
-    <img src="/img/kong-logo.png" alt="Kong logo" />
-    <div class="k-radio-label">Bar</div>
-    <div class="k-radio-description">That subheader</div>
+  <KRadio
+    v-model="cardRadio"
+    description="Choose this option if you want your APIs to only be accessible from within your private network."
+    is-card
+    label="Private"
+    selected-value="private"
+  >
+    <WorldPrivateIcon />
   </KRadio>
   <div>Selected: {{ cardRadio }}</div>
 </template>
 
-<script lang="ts">
-import { defineComponent, reactive, toRefs } from 'vue'
-
-export default defineComponent({
-  setup() {
-    const data = reactive({
-      cardRadio: "",
-    });
+<script setup lang="ts">
+import { ref } from 'vue'
 
-    return {
-      ...toRefs(data),
-    }
-  }
-})
+const cardRadio = ref<string>('')
 </script>
 ```
 
 ### labelAttributes
 
- `KRadio` has an instance of `KLabel` for supporting tooltip text. Use the `labelAttributes` prop to configure the **KLabel's** [props](/components/label). This example shows using the `label-attributes` to set up a tooltip, see the [slot](#slots) section if you want to slot HTML into the tooltip rather than use plain text.
+ KRadio has an instance of KLabel for supporting tooltip text. Use the `labelAttributes` prop to configure the KLabel's [props](/components/label). This example shows using the `labelAttributes` to set up a tooltip. Tooltip content can also be [slotted](#slots).
 
-<KRadio v-model="labelAChecked" label="Tooltips?" :label-attributes="{ help: 'I use the KLabel `help` prop' }" :selected-value="true" />
+<KRadio v-model="labelAttributesPropRadio" label="Tooltips?" :label-attributes="{ info: 'I use the KLabel `help` prop' }" :selected-value="true" />
```

Comment:
issue(maybe): if I click on the tooltip icon, it selects the corresponding radio element. Since users may try clicking on the info icon instead of hovering, should we change this behavior to not select the item on click?